.buildpacks

```diff
@@ -0,0 +1,2 @@
+https://github.com/FabienArcellier/nodejs-buildpack#streamsync-review
```

Comment:
I don't know where to document the role of the Procfile and .buildpacks. ``.buildpacks`` does not support code comment.